main/src/main/scala/sbt/Keys.scala

```diff
@@ -234,6 +234,7 @@ object Keys {
   val classpathOptions = settingKey[ClasspathOptions]("Configures handling of Scala classpaths.").withRank(DSetting)
   val discoveredSbtPlugins = taskKey[PluginDiscovery.DiscoveredNames]("The names of sbt plugin-related modules (modules that extend Build, Plugin, AutoPlugin) defined by this project.").withRank(CTask)
   val sbtPlugin = settingKey[Boolean]("If true, enables adding sbt as a dependency and auto-generation of the plugin descriptor file.").withRank(BMinusSetting)
+  val isMetaBuild = settingKey[Boolean]("If true, this is a metabuild, a project inside project/ directory, and sbt is added to the classpath.").withRank(DSetting)
```

Comment:
How do you choose the ranking for settings?

Reply:
Rank affects how keys show up in help, I think.

Reply:
Ah, okay :laughing: So this is purely UX

Reply:
An opportunity to document that in the code (_hint hint_)